src/qml/controls/InformationPage.qml

```diff
@@ -102,46 +110,9 @@ Page {
             anchors.leftMargin: 20
             anchors.rightMargin: 20
             anchors.horizontalCenter: parent.horizontalCenter
+            anchors.bottom: parent.bottom
             text: root.buttonText
             onClicked: root.lastPage ? swipeView.finished = true : swipeView.incrementCurrentIndex()
         }
     }
-
-    state: AppMode.state
-
-    states: [
-        State {
```

Comment:
The states are required to handle the anchoring differences between mobile and desktop. anchors changes should be handled through States/AnchorChanges. https://doc.qt.io/qt-6/qtquick-positioning-anchors.html#changing-anchors)](https://doc.qt.io/qt-6/qtquick-positioning-anchors.html#changing-anchors